translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"\n"
+"    # Update pod 'foo' by removing an annotation named 'description' if it "
+"exists.\n"
+"    # Does not require the --overwrite flag.\n"
+"    kubectl annotate pods foo description-"
+msgstr ""
+"\n"
+"    # Atualiza o pod ‘foo’ com a annotation ‘description’ e o valor ‘my "
+"frontend’.\n"
+"    # Se a mesma annotation é configurada várias vezes, apenas o último "
+"valor será utilizado\n"
+"    kubectl annotate pods foo description=‘my frontend’\n"
+"\n"
+"    # Atualiza o pod identificado pelo tipo e nome definido no “pod.json”\n"
+"    kubectl annotate -f pod.json description=‘my frontend’\n"
+"\n"
+"    # Atualiza o pod ‘foo’ com a annotation ‘description’ e o valor ‘my "
+"frontend running nginx’, sobreescrevendo qualquer valor existente.\n"
+"    kubectl annotate --overwrite pods foo description=‘my frontend running "
+"nginx’\n"
+"\n"
+"    # Atualiza todos os pods no namespace\n"
+"    kubectl annotate pods --all description=‘my frontend running nginx’\n"
+"\n"
+"    # Atualiza o pod ‘foo’ apenas se o recurso não foi modificado na versão "
+"1.\n"
+"    kubectl annotate pods foo description=‘my frontend running nginx’ --"
+"resource-version=1\n"
+"\n"
+"    # Atualiza o pod ‘foo’ removendo a annotation chamada ‘description’ se "
+"ela existir.\n"
+"    # Não necessita da flag --overwrite.\n"
+"    kubectl annotate pods foo description-"
+
+#: pkg/kubectl/cmd/create_service.go:170
+msgid ""
+"\n"
+"    Create a LoadBalancer service with the specified name."
+msgstr ""
+"\n"
+"    Cria um LoadBalancer service com um nome especificado."
+
+#: pkg/kubectl/cmd/create_service.go:50
+msgid ""
+"\n"
+"    Create a clusterIP service with the specified name."
+msgstr ""
+"\n"
+"    Cria um clusterIP service com um nome especificado."
+
+#: pkg/kubectl/cmd/create_deployment.go:33
+msgid ""
+"\n"
+"    Create a deployment with the specified name."
+msgstr ""
+"\n"
+"    Cria um deployment com um nome especificado."
+
+#: pkg/kubectl/cmd/create_service.go:113
+msgid ""
+"\n"
+"    Create a nodeport service with the specified name."
+msgstr ""
+"\n"
+"    Cria um nodeport service com um nome especificado."
+
+#: pkg/kubectl/cmd/clusterinfo_dump.go:53
+msgid ""
+"\n"
+"    Dumps cluster info out suitable for debugging and diagnosing cluster "
+"problems.  By default, dumps everything to\n"
+"    stdout. You can optionally specify a directory with --output-directory.  "
+"If you specify a directory, kubernetes will\n"
+"    build a set of files in that directory.  By default only dumps things in "
+"the 'kube-system' namespace, but you can\n"
+"    switch to a different namespace with the --namespaces flag, or specify --"
+"all-namespaces to dump all namespaces.\n"
+"\n"
+"    The command also dumps the logs of all of the pods in the cluster, these "
+"logs are dumped into different directories\n"
+"    based on namespace and pod name."
+msgstr ""
+"\n"
+"    Coleta informações do cluster para debugar e diagnosticar problemas do "
```

Comment:
the word 'cluster' is repeated here. Could be something like 'problemas dele' in the end, removing the duplicate word.

Also, 'Por padrão exibe toda a informação no stdout' seems better than 'despeja'